### PR TITLE
console renaming

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Console renaming 🥳 No redirects needed for this one!

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: versioned_docs/version-8.2/components/console/manage-organization/delete-account.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ title: Delete your Camunda Platform account
 keywords: [account, delete, cancel]
 ---
 
+[//]: <> (Do we retain Camunda Platform in this instance?)
```
Comment (akeller) [inline versioned_docs/version-8.2/components/console/manage-organization/delete-account.md:7]: Can you follow up with @dlevy-camunda on this example? You may need to ping him on Slack.

File: versioned_docs/version-8.2/components/console/manage-organization/external-sso.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ description: "For enterprise customers, we support integrating external identity
 
 ## Connecting your identity provider with Camunda Platform
 
+[//]: <> (Similar instance here.)
```
Comment (akeller) [inline versioned_docs/version-8.2/components/console/manage-organization/external-sso.md:10]: Ok to drop it here or lowercase "platform". Your choice.

File: versioned_docs/version-8.2/components/console/manage-organization/delete-account.md (hunk 1 above)
Comment (christinaausley) [reply versioned_docs/version-8.2/components/console/manage-organization/delete-account.md:7]: To delete your Camunda account, confirmed 👍

File: versioned_docs/version-8.2/components/operate/userguide/resolve-incidents-update-variables.md [hunk 3]
```diff
@@ -119,8 +119,6 @@ We’ll publish a message that will be correlated with the instance, so we can a
 
 In the Operate interface, you should now see the process instance has an <!-- FIXME: [“Incident”](/reference/incidents.html) --> incident, which means there’s a problem with process execution that must be fixed before the process instance can progress to the next step.
```
Comment (akeller) [inline versioned_docs/version-8.2/components/operate/userguide/resolve-incidents-update-variables.md:120]: LOL what is this?! 😆 Can you follow up with the Operate team on a separate PR?

**Review by akeller — APPROVED** [review]

File: versioned_docs/version-8.2/components/operate/userguide/resolve-incidents-update-variables.md (hunk 3 above)
Comment (christinaausley) [reply versioned_docs/version-8.2/components/operate/userguide/resolve-incidents-update-variables.md:120]: Lol yes. Will launch a separate issue 👍

**Pull request merged**